firmware/arduino/chessboard/uart.ino

```diff
@@ -1,84 +1,92 @@
 char incomingByte;
 char  buffer[5];
-int byteNum = -1; //-1 indicates that the start code hasn't been received
+int byteNum = -1; // -1 indicates that the start code hasn't been received
 
-//Wait for input
+// Wait for input
 void serialEvent2()
 {
-    //Loop through all available bytes
+    // Loop through all available bytes
     while (Serial2.available())
     {
-        //Get byte
+        // Get byte
         incomingByte = (char) Serial2.read();
 
-        //Reset buffer position
+        // Reset buffer position
         if (incomingByte == '~')
         {
             byteNum = 0;
         }
-        //Add byte to buffer
+        // Add byte to buffer
         else if (byteNum != -1)
         {
             buffer[byteNum++] = incomingByte;
         }
 
-        //Check if the buffer is full, process input
+        // Check if the buffer is full, process input
         if (byteNum == 5)
         {
-            //Reset buffer position
+            // Reset buffer position
             byteNum = -1;
 
-            //Process input
-            //Returns true for valid input and false for invalid input. Calls movement function
+            // Process input
+            // Returns true for valid input and false for invalid input. Calls movement function
             parseMessageFromPi(buffer);
         }
     }
 }
 
 bool parseMessageFromPi(char * message)
 {
-    //Move types 0 and 3 are valid for the same range of values
+    // Move types 0 and 3 are valid for the same range of values
     if (message[0] == 0 || message[0] == 3)
```

Comment:
So this is a change I want you to make for all of these, but when you see any comparator operators, like ==, && and ||, use two spaces between the operator and the arguments. For example...
Good  if (boolean1  &&  boolean2)
Bad     if (boolean1 && boolean2)

Reply:
It just makes it slightly easier to read